src/textual/widgets/_data_table.py

```diff
@@ -14,23 +14,27 @@
 
 from .. import events, messages
 from .._cache import LRUCache
-from ..coordinate import Coordinate
 from .._segment_tools import line_crop
 from .._types import SegmentLines
+from .._typing import Literal
 from ..binding import Binding
+from ..coordinate import Coordinate
 from ..geometry import Region, Size, Spacing, clamp
 from ..message import Message
 from ..reactive import Reactive
 from ..render import measure
 from ..scroll_view import ScrollView
 from ..strip import Strip
-from .._typing import Literal
 
 CursorType = Literal["cell", "row", "column", "none"]
 CELL: CursorType = "cell"
 CellType = TypeVar("CellType")
 
 
+class CellDoesntExist(Exception):
```

Comment:
```suggestion
class CellDoesNotExist(Exception):
```

For some reason, feels wrong to use a contraction in a class name.

Reply:
Yeah, I think this is better. I'll update this.